docs/EXAMPLES.md

```diff
@@ -66,11 +66,35 @@ describe("foo", () => {
 });
 ```
 
+## waitForAlertToBePresent
+
+| 1 optional argument                     |
+|-----------------------------------------|
+| `timeoutInMilliseconds` (default: 5000) |
+
+```js
+const protractorHelper = require("protractor-helper");
+
+describe("foo", () => {
+  it("bar", () => {
+    browser.get("https://example.com");
+
+    const buttonThatTriggersAnAlert = element(by.css('button.some-button'));
+
+    helper.click(buttonThatTriggersAnAlert);
```

Comment:
```suggestion
    protractorHelper.click(buttonThatTriggersAnAlert);
```

Oh gosh, I just noticed that. Sorry for not finding it before. 🤦🏻‍♂️ 

On line 76, we're importing protractor-helper as `protractorHelper`, and so, this is the variable that should be used for the `click` too.